packages/core/jest.config.js

```diff
@@ -6,195 +6,33 @@
 const JEST_END_FILES = process.env.TEST_END_FILES || "jest";
```

Comment:
can probably be removed? alongside the `test:stories` command, as we do not use it